inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/metric/MetricOption.java

```diff
@@ -34,31 +40,33 @@ public class MetricOption {
             + "3}|65[0-4]\\d{"
             + "2}|655[0-2]\\d|6553[0-5])$";
 
-    private String groupId;
-    private String streamId;
-    private String nodeId;
+    private Map<String, String> labels;
     private final HashSet<String> ipPortList;
     private String ipPorts;
     private RegisteredMetric registeredMetric;
 
     private MetricOption(
-            String inlongGroupStreamNode,
+            String inlongLabels,
             @Nullable String inlongAudit,
             @Nullable RegisteredMetric registeredMetric) {
-        Preconditions.checkNotNull(inlongGroupStreamNode,
-                "Inlong group stream node must be set for register metric.");
-        if (inlongGroupStreamNode != null) {
-            String[] inLongGroupStreamNodeArray = inlongGroupStreamNode.split(DELIMITER);
-            Preconditions.checkArgument(inLongGroupStreamNodeArray.length == 3,
-                    "Error inLong metric format: " + inlongGroupStreamNode);
-            this.groupId = inLongGroupStreamNodeArray[0];
-            this.streamId = inLongGroupStreamNodeArray[1];
-            this.nodeId = inLongGroupStreamNodeArray[2];
-        }
+        Preconditions.checkArgument(!StringUtils.isNullOrWhitespaceOnly(inlongLabels),
+                "Inlong labels must be set for register metric.");
+
+        this.labels = new HashMap<>();

```

Comment:
It should use LinkHashMap, keep full metric name order

Reply:
OK